x/evm/types/params.go

```diff
@@ -128,6 +130,16 @@ func (p Params) GetActivePrecompilesAddrs() []common.Address {
 	return precompiles
 }
 
+// IsPrecompileRegistered returns true if the given precompile address is
+// registered as an active precompile.
+func (p Params) IsPrecompileRegistered(address string) bool {
+	_, found := sort.Find(len(p.ActivePrecompiles), func(i int) int {
+		return strings.Compare(address, p.ActivePrecompiles[i])
+	})
```

Comment:
I think the `Find` method requires the slice of strings to be sorted, do we make sure of that when storing the active precompiles? 🤔

Reply:
ahh just found where this was added on this PR - I'd still add a test case specifically to check this